libkirk/ltp.py

```diff
@@ -384,6 +385,8 @@ async def read_result(
                     warnings = 1
                 elif retcode == 32:
                     skipped = 1
+                elif retcode == -signal.SIGKILL:
+                    skipped = 1
```

Comment:
I'm not 100% sure that we should do this unconditionally. It would be much safer to do this only if the force stop was in progress, but to do that we would have to propagate the force stop status from the session here.